components/contribution-flow/ContributionSummary.js

```diff
@@ -64,9 +64,9 @@ const ContributionSummary = ({
 }) => {
   const intl = useIntl();
   const amount = isCrypto ? stepDetails.cryptoAmount : stepDetails.amount;
-  const totalAmount = getTotalAmount(stepDetails, stepSummary);
+  const totalAmount = getTotalAmount(stepDetails, stepSummary, isCrypto);
   const pmFeeInfo = getPaymentMethodFees(stepPayment?.paymentMethod, totalAmount, currency);
-  const platformTip = stepDetails.platformTip || 0;
+  const platformTip = !isCrypto ? stepDetails.platformTip || 0 : 0;
```

Comment:
Weird to read

Reply:
I've used `lodash` to simplify a bit. Let me know.